bin/varnishd/http2/cache_http2_send.c

```diff
@@ -41,6 +41,41 @@
 
 #define H2_SEND_HELD(h2, r2) (VTAILQ_FIRST(&(h2)->txqueue) == (r2))
 
+static int
+h2_cond_wait(pthread_cond_t *cond, struct h2_sess *h2, struct h2_req *r2)
+{
+	vtim_real now, tmo;
+	int r;
+
+	AN(cond);
+	CHECK_OBJ_NOTNULL(h2, H2_SESS_MAGIC);
+	CHECK_OBJ_NOTNULL(r2, H2_REQ_MAGIC);
+
+	now = VTIM_real();
+	tmo = cache_param->idle_send_timeout;
```

Comment:
this does not seem to make much sense to me. We should still enforce `send_timeout` for the entirety of the response, but I guess this is incomplete because you are just trying to make progress?
If yes, mark this maybe?
(side note: `idle_send_timeout` might not be the best choice of name - rename maybe to `unacknowledged_send_timeout`?  )

Reply:
You are right, the overall `send_timeout` is treated like `idle_send_timeout` in my patch. I will try to fix this today, good catch.

Reply:
Fixed in 980f0f5df35a34212e3f7a0ac8890209fafbc511.